en/flight_controller/cubepilot_cube_orange.md

```diff
@@ -108,16 +108,16 @@ Board schematics and other documentation can be found here: [The Cube Project](h
 UART | Device | Port
 --- | --- | ---
 USART1 | /dev/ttyS0 | <!-- # IO DEBUG:/dev/ttyS0 -->
-USART2 | /dev/ttyS1 | TELEM1 (flow control)
-USART3 | /dev/ttyS2 | TELEM2 (flow control)
+USART2 | /dev/ttyS1 | TELEM2 (flow control)
+USART3 | /dev/ttyS2 | TELEM1 (flow control)
 UART4  | /dev/ttyS3 | GPS1
```

Comment:
@AjayaDahal I think this is incorrect - it should be:

UART | Device | Port
--- | --- | ---
USART1 | /dev/ttyS0 | TELEM1 (flow control)
USART2 | /dev/ttyS1 | TELEM2 (flow control)
USART3 | /dev/ttyS2 | GPS1
UART4  | /dev/ttyS3 | PX4IO
USART6 | /dev/ttyS4 | 
UART7  | /dev/ttyS5 | CONSOLE
UART8  | /dev/ttyS6 | GPS2

This is based on the process outlined here: https://github.com/PX4/PX4-user_guide/pull/672#issuecomment-598198434

Can you confirm - my working below:

----

For which I get:

https://github.com/PX4/PX4-Autopilot/blob/main/boards/cubepilot/cubeorange/default.px4board#L4-L7

CONFIG_BOARD_SERIAL_GPS1="/dev/ttyS2"
CONFIG_BOARD_SERIAL_GPS2="/dev/ttyS5"
CONFIG_BOARD_SERIAL_TEL1="/dev/ttyS0"
CONFIG_BOARD_SERIAL_TEL2="/dev/ttyS1"

https://github.com/PX4/PX4-Autopilot/blob/main/boards/cubepilot/cubeorange/nuttx-config/nsh/defconfig#L188-L197

ttyS0 CONFIG_STM32H7_USART2=y = TEL1
ttyS1 CONFIG_STM32H7_USART3=y = TEL2
ttyS2 CONFIG_STM32H7_UART4=y = GPS1
ttyS3 CONFIG_STM32H7_USART6=y
ttyS4 CONFIG_STM32H7_UART7=y
ttyS5 CONFIG_STM32H7_UART8=y = GPS2


CONFIG_DEV_CONSOLE - not set

https://github.com/PX4/PX4-Autopilot/blob/main/boards/cubepilot/cubeorange/src/board_config.h#L49

#define PX4IO_SERIAL_DEVICE            "/dev/ttyS3" = USART6